Comparing F:\Luna_skola_workspace2\testing-lib\testing-lib-v2\target\testing-lib-v2-0.0.1.jar with F:\Luna_skola_workspace2\testing-lib\testing-lib-v2\target\testing-lib-v2-0.0.2.jar:
---! REMOVED CLASS: PUBLIC(-) org.testing.lib.v1.App  (not serializable)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) App()
	---! REMOVED METHOD: PUBLIC(-) STATIC(-) void main(java.lang.String[])
+++  NEW CLASS: PUBLIC(+) org.testing.lib.v2.App  (not serializable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) App()
	+++  NEW METHOD: PUBLIC(+) STATIC(+) void main(java.lang.String[])
***  MODIFIED CLASS: PUBLIC NON_ABSTRACT (<- ABSTRACT) testing_lib.clazz.clazzAbstractToNonAbstract.ClazzAbstractToNonAbstract  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzAbstractToNonAbstract()
***  MODIFIED CLASS: PUBLIC NON_FINAL (<- FINAL) testing_lib.clazz.clazzFinalToNonFinal.ClazzFinalToNonFinal  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzFinalToNonFinal()
***! MODIFIED CLASS: PUBLIC ABSTRACT (<- NON_ABSTRACT) testing_lib.clazz.clazzNonAbstractToAbstract.ClazzAbstractToNonAbstract  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzAbstractToNonAbstract()
***! MODIFIED CLASS: PUBLIC FINAL (<- NON_FINAL) testing_lib.clazz.clazzNonFinalToFinal.ClazzFinalToNonFinal  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzFinalToNonFinal()
***! MODIFIED INTERFACE (<- CLASS) : PUBLIC ABSTRACT (<- NON_ABSTRACT) testing_lib.clazz.clazzToIfaze.ClazzToIfaze  (not serializable)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) ClazzToIfaze()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.constructorAdd.ClazzConstructorAdd  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzConstructorAdd()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.constructorDelete.ClazzConstructorDelete  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzConstructorDelete()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.contractSuperClassSet.Clazz1  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC Clazz1()
---! REMOVED CLASS: PUBLIC(-) testing_lib.clazz.contractSuperClassSet.Clazz2  (not serializable)
	---! REMOVED CONSTRUCTOR: PUBLIC(-) Clazz2()
***! MODIFIED CLASS: PUBLIC testing_lib.clazz.contractSuperClassSet.ClazzExpandSuperInterfaceSet  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzExpandSuperInterfaceSet()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.expandSuperClassSet.Clazz1  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC Clazz1()
+++  NEW CLASS: PUBLIC(+) testing_lib.clazz.expandSuperClassSet.Clazz2  (not serializable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) Clazz2()
***! MODIFIED CLASS: PUBLIC testing_lib.clazz.expandSuperClassSet.ClazzExpandSuperClassSet  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzExpandSuperClassSet()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.fieldAdd.ClazzFieldAdd  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzFieldAdd()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.fieldDelete.ClazzFieldDelete  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzFieldDelete()
===  UNCHANGED CLASS: PUBLIC ABSTRACT testing_lib.clazz.method.methodAbstractToNonAbstract.ClazzMethodAbstractToNonAbstract  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodAbstractToNonAbstract()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.methodFinalToNonFinal.ClazzMethodFinalToNonFinal  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodFinalToNonFinal()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.methodNativeToNonNative.ClazzMethodNativeToNonNative  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodNativeToNonNative()
===  UNCHANGED CLASS: PUBLIC ABSTRACT testing_lib.clazz.method.methodNonAbstractToAbstract.ClazzMethodNonAbstractToAbstract  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodNonAbstractToAbstract()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.methodNonFinalToFinal.ClazzMethodNonFinalToFinal  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodNonFinalToFinal()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.methodNonNativeToNative.ClazzMethodNonNativeToNative  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodNonNativeToNative()
===  UNCHANGED CLASS: PUBLIC ABSTRACT testing_lib.clazz.method.methodNonStaticToStatic.ClazzMethodNonStaticToStatic  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodNonStaticToStatic()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.methodNonSynchronizedToSynchronized.ClazzMethodSynchronizedToNonSynchronized  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodSynchronizedToNonSynchronized()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.methodRename.ClazzMethodNameChange  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodNameChange()
===  UNCHANGED CLASS: PUBLIC ABSTRACT testing_lib.clazz.method.methodStaticToNonStatic.ClazzMethodStaticToNonStatic  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodStaticToNonStatic()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.methodSynchronizedToNonSynchronized.ClazzMethodSynchronizedToNonSynchronized  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodSynchronizedToNonSynchronized()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.clazz.method.methodTypeDownHierarchy.ClazzMethodTypeDownHierarchy  (not serializable)
	***! MODIFIED METHOD: PUBLIC ABSTRACT java.lang.Integer (<-java.lang.Number) method()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.clazz.method.methodTypeUpHierarchy.ClazzMethodTypeUpHierarchy  (not serializable)
	***! MODIFIED METHOD: PUBLIC ABSTRACT java.lang.Number (<-java.lang.Integer) method()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.paramAddOrDelete.ClazzMethodParamAddOrDelete  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodParamAddOrDelete()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.clazz.method.paramNameChange.ClazzMethodParamNameChange  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void method1(java.lang.Integer)
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.paramTypeChange.arrayToVarArity.ClazzMethodParamTypeChangeLastParamArrayToVarArity  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodParamTypeChangeLastParamArrayToVarArity()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.paramTypeChange.intToString.ClazzMethodParamTypeChangeIntToSting  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodParamTypeChangeIntToSting()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.paramTypeChange.varArityToArray.ClazzMethodParamTypeChangeLastParamVarArityToArray  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodParamTypeChangeLastParamVarArityToArray()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.typeParameterAdd.ClazzMethodTypeParamAdd  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodTypeParamAdd()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.typeParameterBounded.ClazzMethodTypeParamBounded  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodTypeParamBounded()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.typeParameterDelete.ClazzMethodTypeParamDelete  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodTypeParamDelete()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.method.typeParameterPositionChange.ClazzMethodTypeParamPositionChange  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodTypeParamPositionChange()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.methodAdd.ClazzMethodAdd  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodAdd()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.methodDelete.ClazzMethodDelete  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzMethodDelete()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.typeParamAdd.ClazzTypeParamAdd  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzTypeParamAdd()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.typeParamChangeBoundsIntegerToNumber.ClazzTypeParamChangeBoundsIntegerToNumber  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzTypeParamChangeBoundsIntegerToNumber()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.typeParamChangeBoundsNumberToInteger.ClazzTypeParamChangeBoundsNumberToInteger  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzTypeParamChangeBoundsNumberToInteger()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.typeParamDelete.ClazzTypeParamDelete  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzTypeParamDelete()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.typeParamRename.ClazzTypeParamRename  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzTypeParamRename()
===  UNCHANGED CLASS: PUBLIC testing_lib.clazz.typeParamSwap.ClazzTypeParamSwap  (not serializable)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ClazzTypeParamSwap()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.contractSuperinterfaceSet.IfazeExpandSuperinterfaceSet  (not serializable)
	---! REMOVED INTERFACE: testing_lib.ifaze.contractSuperinterfaceSet.Interface2
	+++  NEW INTERFACE: testing_lib.ifaze.contractSuperinterfaceSet.Interface1
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.contractSuperinterfaceSet.Interface1  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void ifaze1method1()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.contractSuperinterfaceSet.Interface2  (not serializable)
	---! REMOVED INTERFACE: testing_lib.ifaze.contractSuperinterfaceSet.Interface1
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void ifaze2method1()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.expandSuperinterfaceSet.IfazeExpandSuperinterfaceSet  (not serializable)
	---! REMOVED INTERFACE: testing_lib.ifaze.expandSuperinterfaceSet.Interface1
	+++  NEW INTERFACE: testing_lib.ifaze.expandSuperinterfaceSet.Interface2
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.expandSuperinterfaceSet.Interface1  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void ifaze1method1()
***  MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.expandSuperinterfaceSet.Interface2  (not serializable)
	+++  NEW INTERFACE: testing_lib.ifaze.expandSuperinterfaceSet.Interface1
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void ifaze2method1()
***  MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.fieldAdd.IfazeFieldAdd  (not serializable)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.Integer field1
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.Integer addedField
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.fieldDelete.IfazeFieldDelete  (not serializable)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.Integer field1
	---! REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) java.lang.Integer fieldToBeRemoved
***! MODIFIED CLASS (<- INTERFACE) : PUBLIC NON_ABSTRACT (<- ABSTRACT) testing_lib.ifaze.ifazeToClass.IfazeToClass  (not serializable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) IfazeToClass()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.methodRename.IfazeMethodNameChange  (not serializable)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void methodNameChanged()
	---! REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void methodNameToBeChanged()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.methodTypeDownHierarchy.IfazeMethodTypeDownHierarchy  (not serializable)
	***! MODIFIED METHOD: PUBLIC ABSTRACT java.lang.Integer (<-java.lang.Number) method()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.methodTypeUpHierarchy.IfazeMethodTypeUpHierarchy  (not serializable)
	***! MODIFIED METHOD: PUBLIC ABSTRACT java.lang.Number (<-java.lang.Integer) method()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.paramAddOrDelete.IfazeMethodParamAddOrDelete  (not serializable)
	---! REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void method1(java.lang.Integer)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void method1(java.lang.Integer, java.lang.Integer)
	---! REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void method2(java.lang.Integer, java.lang.Integer)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void method2(java.lang.Integer)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.paramNameChange.IfazeMethodParamNameChange  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void method1(java.lang.Integer)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.paramTypeChange.arrayToVarArity.IfazeMethodParamTypeChangeLastParamArrayToVarArity  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void method1(java.lang.Integer, java.lang.Object[])
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.paramTypeChange.intToString.IfazeMethodParamTypeChangeIntToString  (not serializable)
	---! REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void method1(int)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void method1(java.lang.String)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.paramTypeChange.varArityToArray.IfazeMethodParamTypeChangeLastParamVarArityToArray  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void method1(java.lang.Integer, java.lang.Object[])
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.returnTypeChange.IfazeMethodReturnTypeChange  (not serializable)
	***! MODIFIED METHOD: PUBLIC ABSTRACT java.lang.String (<-java.lang.Integer) methodIntegerToString()
	***! MODIFIED METHOD: PUBLIC ABSTRACT void (<-java.lang.Integer) methodIntegerToVoid()
	***! MODIFIED METHOD: PUBLIC ABSTRACT java.lang.Integer (<-void) methodVoidToInteger()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.typeParameterAdd.IfazeMethodTypeParamAdd  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void methodToBeParametrized()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.typeParameterBounded.IfazeMethodTypeParamBounded  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void methodNotParametrized()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.typeParameterDelete.IfazeMethodTypeParamDelete  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void methodNotParametrized()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.method.typeParameterPositionChange.IfazeMethodTypeParamPositionChange  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void methodNotParametrized()
***  MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.methodAdd.IfazeMethodAdd  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void method1()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void methodAdded()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.methodDelete.IfazeMethodDelete  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void method1()
	---! REMOVED METHOD: PUBLIC(-) ABSTRACT(-) void methodToBeDeleted()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamAdd.IfazeTypeParamAdd  (not serializable)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamAddSecondParam.IfazeTypeParamAddSecondParam  (not serializable)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamChangeBounds.IfazeTypeParamChangeBounds  (not serializable)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamDelete.IfazeTypeParamDelete  (not serializable)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamDeleteOneParam.IfazeTypeParamDelete  (not serializable)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamDeletePlusMethod.IfazeTypeParamDeletePlusMethod  (not serializable)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void method1(java.lang.Object)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamRename.IfazeTypeParamRename  (not serializable)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT testing_lib.ifaze.typeParamSwap.IfazeTypeParamSwap  (not serializable)
